blivet/devices/disk.py

```diff
@@ -725,7 +725,8 @@ class NVMeFabricsNamespaceDevice(NVMeNamespaceDevice, NetworkStorageDevice):
 
     """ NVMe fabrics namespace """
     _type = "nvme-fabrics"
-    _packages = ["nvme-cli"]
+    # dracut '95nvmf' module dependencies
+    _packages = ["nvme-cli", "dracut-network", "jq"]
```

Comment:
`jq` sounds like a really weird dependency for us to install

Reply:
Agree, this looks out of scope of blivet, though I haven't found a better place. This would perhaps need to be somewhere in Anaconda otherwise.